ivy/functional/frontends/numpy/manipulation_routines/changing_kind_of_array.py

```diff
@@ -0,0 +1,12 @@
+# local
+import ivy
+
+
+def asarray(obj,
+            dtype=None,
+            order=None,
+            *,
+            like=None):
+    if dtype:
```

Comment:
I'm pretty sure `ivy.asarray` can handle `dtype` args, what is the reason for this extra type handling?

Reply:
Well, I'll remove this logic duplicate